File: app_manager/appmanager/src/main/java/app/editors/manager/mvp/views/login/PersonalRegisterView.java
package app.editors.manager.mvp.views.login;

import moxy.viewstate.strategy.OneExecutionStateStrategy;
import moxy.viewstate.strategy.StateStrategyType;

import app.editors.manager.mvp.views.base.BaseView;

@StateStrategyType(OneExecutionStateStrategy.class)
public interface PersonalRegisterView extends BaseView {
    void onRegisterPortal();
}

File: app_manager/appmanager/src/main/java/app/editors/manager/mvp/views/login/PersonalRegisterView.kt
package app.editors.manager.mvp.views.login

import androidx.annotation.StringRes
import app.editors.manager.mvp.views.base.BaseView
import moxy.viewstate.strategy.OneExecutionStateStrategy
import moxy.viewstate.strategy.StateStrategyType

@StateStrategyType(OneExecutionStateStrategy::class)
interface PersonalRegisterView : BaseView {
    fun onRegisterPortal()
    fun onWaitingDialog()
    fun onMessage(@StringRes message: Int)
}